docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -192,3 +192,36 @@ keep the Password field empty:
 ![DataGrip connection options](../../../img/database-access/guides/sqlserver/datagrip-connection@2x.png)
 
 Click OK to connect.
+
+## Redis Insight
+
+<Admonition type="note">
+  Teleport's Redis Insight integration only supports Redis standalone instances.
+</Admonition>
+
+After opening Redis Insight click `ADD REDIS DATABASE`.
+
+![Redis Insight Startup Screen](../../../img/database-access/guides/redis/redisinsight-startup.png)
+
+Click `Add Database Manually`. Use `127.0.0.1` as the `Host` and port printed by `tsh proxy db`.
+
+<Admonition type="note">
```

Comment:
Should this Admonition be regular text that comes before the "Click..." instruction above? Otherwise, readers will need to follow these instructions out of order.

Reply:
I'll move it.